extensions/opentelemetry/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/tracing/TracerProcessor.java

```diff
@@ -60,7 +61,8 @@ static class MetricsExtensionAvailable implements BooleanSupplier {
 
         @Override
         public boolean getAsBoolean() {
-            return IS_MICROMETER_EXTENSION_AVAILABLE;
+            return IS_MICROMETER_EXTENSION_AVAILABLE && ConfigProvider.getConfig()
+                    .getOptionalValue("quarkus.micrometer.binder.http-server.enabled", Boolean.class).orElse(true);
```

Comment:
@radcortez Found an issue here when testing this:
The default for [quarkus.micrometer.binder.http-server.enabled](https://quarkus.io/guides/micrometer#quarkus-micrometer_quarkus.micrometer.binder.http-server.enabled) is not necessarily `true`.

If `quarkus.micrometer.binder-enabled-default` is `false`, then `quarkus.micrometer.binder.http-server.enabled` is also `false` by default.

Reply:
Ok, I'll check.

Reply:
This should be the proper check:
https://github.com/quarkusio/quarkus/blob/8816386b87e145ade6ef55aa0d7572f20c6eec21/extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/config/MicrometerConfig.java#L70-L80

Reply:
In https://github.com/quarkusio/quarkus/pull/26871.